cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3580,7 +3580,6 @@ func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedAbsentDataS
 	defer restore()
 
 	dataActivated := false
-	saveActivated := false
```

Comment:
the name of this test now is misleading, the name of the test should now be something like
```
TestInitramfsMountsRecoverModeDegradedAbsentDataAbsentSaveHappy
```

Reply:
Renamed to `TestInitramfsMountsRecoverModeDegradedAbsentDataUnencryptedSaveHappy` since the scenario uses a plain save partition.